ang/civicase/activity/feed/directives/activity-feed.directive.html

```diff
@@ -117,10 +116,10 @@
       <!-- end activity details -->
       <!-- quick month nav -->
       <div
+        ng-show='isMonthNavVisible'
```

Comment:
Do you think it's possible to use `ng-if` instead of `ng-show` ?

Reply:
I tried using `ng-if` but its not rendering the directive at all. I tried googling but could not find anything. Already spent some time on this. So lets have a chat when you are online.

Reply:
It's ok, this is not critical.